spec/unit/coinbase/webhook_spec.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+describe Coinbase::Webhook do
+  let(:api_client) { instance_double(Coinbase::Client::ApiClient) }
+  let(:webhooks_api) { instance_double(Coinbase::Client::WebhooksApi) }
+  let(:webhook_model) do
+    Coinbase::Client::Webhook.new(
```

Comment:
Can we fast follow and add some factories for this client object?

Reply:
sure, will log a ticket to track this